data/xsd/OspSystemStructure.xsd

```diff
@@ -16,6 +16,7 @@
                 <xs:element name="Simulators" type="osp:simulators"/>
                 <xs:element name="Functions" type="osp:functions" minOccurs="0"/>
                 <xs:element name="Connections" type="osp:connections" minOccurs="0"/>
+                <xs:element name="EccoConfiguration" type="osp:eccoconfiguration" minOccurs="0"/>
```

Comment:
need to specify maxOccurs?

Reply:
Good idea. In fact, it might be a good idea to also specify `maxOccurs` for `Algorithm`, `BaseStepSize`, `StartTime `and `EndTime`. So I did just that.